ivy_tests/test_ivy/test_functional/test_extensions.py

```diff
@@ -165,8 +165,7 @@ def test_sinc(
 @given(
     dtype_and_x=helpers.dtype_and_values(
         available_dtypes=helpers.get_dtypes("float", full=False),
-        min_num_dims=1,
-        max_num_dims=1,
+        min_num_dims=1, max_num_dims=1
```

Comment:
Please remove this line and stick to the previous standard of representing function arguments, with one argument on one line, ended by a comma.